gensim/models/fasttext.py

```diff
@@ -41,7 +121,63 @@
     >>> fname = get_tmpfile("fasttext.model")
     >>>
     >>> model.save(fname)
-    >>> model = FastText.load(fname)  # you can continue training with the loaded model!
+    >>> model = FastText.load(fname)
+
+Once loaded, such models behave identically to those created from scratch.
+For example, you can continue training the loaded model:
+
+.. sourcecode:: pycon
+
+    >>> import numpy as np
+    >>> 'computation' in model.wv.vocab  # New word, currently out of vocab
+    False
+    >>> old_vector = np.copy(model.wv['computation'])  # Grab the existing vector
+    >>> new_sentences = [
+    ...     ['computer', 'aided', 'design'],
+    ...     ['computer', 'science'],
+    ...     ['computational', 'complexity'],
+    ...     ['military', 'supercomputer'],
+    ...     ['central', 'processing', 'unit'],
+    ...     ['onboard', 'car', 'computer'],
+    ... ]
+    >>> model.build_vocab(new_sentences, update=True)  # Update the vocabulary
+    >>> model.train(new_sentences, total_examples=len(new_sentences), epochs=model.epochs)
+    >>> new_vector = model.wv['computation']
+    >>> np.allclose(old_vector, new_vector, atol=1e-4)  # Vector has changed, model has learnt something
+    False
+    >>> 'computation' in model.wv.vocab  # Word is still out of vocab
+    False
+
+.. Important::
+    Be sure to call the :meth:`~gensim.models.fasttext.FastText.build_vocab`
```

Comment:
deserve a description, why this needed?

Reply:
Added

Reply:
missing mention about `update_vocab=True`

Reply:
OK, added this too